etc/kayobe/kolla/globals.yml

```diff
@@ -1,7 +1,11 @@
 # yamllint disable-file
 ---
-docker_yum_baseurl: "{{ stackhpc_repo_docker_url }}"
-docker_yum_gpgkey: "https://download.docker.com/linux/centos/gpg"
+
+# To work around issue of trying to install docker from 
+# empty pulp server, use upstream docker dnf repo on 
+# non-overcloud hosts
+enable_docker_repo: {% raw %}"{{ 'overcloud' not in group_names }}"{% endraw %}
```

Comment:
Oops, looks like this broke Ubuntu: https://github.com/stackhpc/stackhpc-kayobe-config/actions/runs/3677794367/jobs/6220373647

Perhaps it also needs to be enabled when `ansible_facts.os_family == 'Debian'`

Reply:
(This is because we don't use release train for host packages on Ubuntu)

Reply:
https://github.com/stackhpc/stackhpc-kayobe-config/pull/295

Reply:
Merged